integration_tests/test_load_connection.py

```diff
@@ -34,7 +34,7 @@ def on_market_cycle(self, market_info):
                 assert listed_bid["energy"] == bid_info["energy"]
                 # Try to delete the bid
                 delete_resp = self.delete_bid(bid_info["id"])
-                assert delete_resp["bid_deleted"] == bid_info["id"]
+                assert delete_resp["bids_deleted"] == [bid_info["id"]]
```

Comment:
I do not have an objection here, but I think it would be nice for Colin to know about this change, so that he adapts his test scripts to this change.

Reply:
ok! Will do, especially after incorporating @faizan2590 request, which i agree on.